src/send/SendSelectRecipient.tsx

```diff
@@ -130,34 +126,123 @@ const getStartedStyles = StyleSheet.create({
   },
 })
 
-function SendSelectRecipient() {
+function SendSelectRecipient({ route }: Props) {
   const { t } = useTranslation()
+  const dispatch = useDispatch()
 
-  const [searchQuery, setSearchQuery] = useState('')
-  const throttledSearch = throttle((searchInput: string) => {
-    setSearchQuery(searchInput)
-  }, SEARCH_THROTTLE_TIME)
+  const forceTokenId = route.params?.forceTokenId
+  const defaultTokenIdOverride = route.params?.defaultTokenIdOverride
 
+  const [sendOrInviteButtonHidden, setSendOrInviteButtonHidden] = useState(true)
+  const [showInviteModal, setShowInviteModal] = useState(false)
   const [showContacts, setShowContacts] = useState(false)
+  const [showSearchResults, setShowSearchResults] = useState(false)
```

Comment:
Invite doesn't easily slot in here since it's an "overlay" over either the search or contacts screen. Get Started/Recents can be merged into a single state, since "get started" turns into "recents" when `recentRecipients` isn't empty, but I don't think we want to do something like call `setActiveView('recent')` inside of a useMemo when `recentRecipients` changes. I've boiled it down to `get started`, `contacts`, and `search` however and it does clean things up a little bit.